client/src/components/questionsForm.tsx

```diff
@@ -1,20 +1,21 @@
 import React, { useState } from "react";
-import { models } from "../../../common";
+import type { models } from "../../../common";
 
-const QuestionsForm: React.FC<{
-	onAddQuestion: (question: { title: string }) => Promise<void>;
-}> = ({ onAddQuestion }) => {
+interface QuestionsFromProps {
+	onAddQuestion: (question: models.Question) => Promise<void>;
+}
+type ErrorValues = Partial<Record<keyof models.Question, string>>;
```

Comment:
This was used twice, and hence I extracted. I did not move out to another file since there is no further usage currently